.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,59 @@
+name: E2E Testing for CLI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [synchronize, opened, reopened, edited]
+    branches: [main]
+  schedule:
+    # check daily to notice potential package manager issues
+    - cron: '0 1 * * *' # At 01:00 daily
+
+jobs:
+  e2e:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 10
+    env:
+      DEBUG: true
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+        node-version: ['12.x', '*']
+        package-manager: ['npm', 'yarn', 'pnpm']
```

Comment:
Could we move the `package-manager` inside the test suite?
I'm concerned this will take up quite a few CI machines.

Since we install the CLI locally we can create a directory for each package manager and install the CLI locally to that directory.

Reply:
Yea no problem just wanted to achieve the fastest result :D